README.md

```diff
@@ -94,6 +85,16 @@ Optionally you can pass the logger a set of configs. Each config type that maps
     }
     ```
 
+- `params`: Configuration that applies to all logger methods:
+
+  - `disable` *[Array of Strings]*: Disable all logging in certain environments. Each entry can be either an environment variable or the value of the `NODE_ENV` environment variable.
+  - Default: `[]`.
+    - Example usage:
+      - `['LOADED_MOCHA_OPTS']`: Disables logger when app is being run by [Mocha](https://mochajs.org/).
+      - `['production']`: Disables logger when `NODE_ENV` is set to `production`.
+
+  - `enablePrefix` *[Boolean]*: Enable prefixes which can contain emojis or other strings to be prepended to logs. This can also be toggled with the `ROOSEVELT_LOGGER_ENABLE_PREFIX` environment variable.
```

Comment:
this should be above default

Reply:
Actually it looks like the `Default` bullet is default options for the `disable` param so I think it should be indented